src/core_plugins/kibana/server/lib/export/collect_panels.js

```diff
@@ -0,0 +1,30 @@
+import { get } from 'lodash';
+
+import collectIndexPatterns from './collect_index_patterns';
+import collectSearchSources from './collect_search_sources';
+
+export const deps = {
+  collectSearchSources,
+  collectIndexPatterns
+};
+
+export default function collectPanels(savedObjectsClient, dashboard) {
```

Comment:
If this was an async function you would benefit from returning a rejected promise if `JSON.parse()` fails, and probably wouldn't have forgotten to return the promise on line 15.